dataframe/arrow.go

```diff
@@ -103,34 +103,34 @@ func buildArrowColumns(f *Frame, arrowFields []arrow.Field) ([]array.Column, err
 	for fieldIdx, field := range f.Fields {
 		switch v := field.Vector.(type) {
 
-		case *intVector:
+		case *Int64Vector:
```

Comment:
Do these need to be exported? Or is did it just get capitalized in renaming?

Reply:
I did try changing these in `generic_vector.go` to be `type genericVector struct`. However, when I run `go generate` with this change the output in `vector.gen.go` is all messed up. There are repeated declarations of `genericVector` in the file. 

I'm guessing there is something I missing with genny. Either something obvious in the `//go:generate` line, or maybe something more complicated like a collision with using that name?

Reply:
This is automatically named from `GenericType` by replacing the `Generic` with each type. Off course I could add a prefix so it will not be exported (just like what happens with the nullable types). Any prefix suggestions?

Reply:
> I did try changing these in `generic_vector.go` to be `type genericVector struct`. However, when I run `go generate` with this change the output in `vector.gen.go` is all messed up. There are repeated declarations of `genericVector` in the file.
> 
> I'm guessing there is something I missing with genny. Either something obvious in the `//go:generate` line, or maybe something more complicated like a collision with using that name?

maybe this would work:
`//go:generate genny -in=$GOFILE -out=vector.gen.go gen "Generic=uint8,uint16,uint32,uint64,int8,int16,int32,int64,float32,float64,string,bool,time.Time generic=uint8,uint16,uint32,uint64,int8,int16,int32,int64,float32,float64,string,bool,time.Time"`

Reply:
Think I got it. change `Generic` to `g` and elsewhere. Will make a change in another branch to demonstrate a little bit later after meeting.

Reply:
I had tried `generic`, but it I don't think it like the import rename on the package `generic`. But maybe I typed that line. `g` seems to work though.

Reply:
see https://github.com/grafana/grafana-plugin-sdk-go/pull/33